modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphologyrelease/v0.1.1.json

```diff
@@ -33,8 +33,7 @@
               "path": "nsg:morphologyIndex",
               "name": "Morphology release index file",
               "description": "Morphology release index file.",
-              "seeAlso": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape",
-              "class": "nsg:Entity"
+              "node": "{{base}}/schemas/nexus/core/distribution/v0.1.0/shapes/DistributionShape"
```

Comment:
I would not recommend this approach. Instead, submit the morphologyIndex as an entity (so against the schema [neurosciencegraph/core/entity/v0.1.0.json](https://github.com/INCF/neuroshapes/blob/master/modules/core/src/main/resources/schemas/neurosciencegraph/core/entity/v0.1.0.json)) and then refer to it here as nsg:morphologyIndex value.

The shape could be:
```
{
              "path": "nsg:morphologyIndex",
              "name": "Morphology release index file",
              "description": "Morphology release index file.",
              "class":"nsg:Entity", (or nsg:MorphologyIndex is you want a specific type)
              "seeAlso": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
}
```

Reply:
Done.